clang/include/clang/AST/OperationKinds.def

```diff
@@ -361,6 +361,9 @@ CAST_OPERATION(AddressSpaceConversion)
 // Convert an integer initializer to an OpenCL sampler.
 CAST_OPERATION(IntToOCLSampler)
 
+// Truncate a vector type (HLSL only).
+CAST_OPERATION(VectorTruncation)
```

Comment:
Should we name this `HLSLVectorTruncation` so it's especially clear this only applies to HLSL?